<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialMonitor</class>
 <widget class="QWidget" name="SerialMonitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="RawData_GroupBox">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>Raw Data</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="Gyro_radioButton">
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Gyroscope</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Accel_radioButton">
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Accelerometer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Mag_radioButton">
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Magnetometer</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="CalibratedData_GroupBox">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>Filtered Data</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="CalGyro_radioButton">
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>GyroScope</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="CalAccel_radioBUtton">
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Accelerometer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="CalMag_radioButton">
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Magnetometer</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="Angles_GroupBox">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>Angles</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="Eulers_radioButton">
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Euler's Angle</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Quaternion_radioButton">
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Quaternion</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Send_Button">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Clear_Button">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Start_Button">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
